saleor/graphql/product/tests/test_product.py

```diff
@@ -458,10 +456,10 @@ def test_products_query_with_filter_collection(
 @pytest.mark.parametrize(
     "products_filter",
     [
-        {"price": {"gte": 5.0, "lte": 9.0}},
         {"minimalPrice": {"gte": 1.0, "lte": 2.0}},
         {"isPublished": False},
         {"search": "Juice1"},
+        {"price": {"gte": 1.0, "lte": 2.0}},
```

Comment:
If you restore some code please do it without changing line order. It's easier to do a review without looking at the changes that "change nothing".